package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build",
+    "build": "tsc && NODE_OPTIONS=--max-old-space-size=8192 vite build",
```

Comment:
This will only fix the memory issue for the local build. The deployment build already has memory size set to 8192 mb.